.github/workflows/deploy-RELEASE.yml

```
@@ -0,0 +1,70 @@
+---
+#########################
+#########################
+## Deploy Docker Image ##
+#########################
+#########################
+
+#
+# Documentation:
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+#
+
+#######################################
+# Start the job on all push to master #
+#######################################
+on:
+  release:
+    # Want to run the automation when a release is created
+    types: ['created']
```

This should fix our issue where the deploy script gets kicked off every time that the release notes drafter did things
```suggestion
    types: ['published']
```